package.json

```diff
@@ -21,7 +21,14 @@
     "lint": "dtslint types"
   },
   "devDependencies": {
+    "@types/node": "^8.0.53",
     "dtslint": "github:Microsoft/dtslint#production",
-    "types-publisher": "Microsoft/types-publisher#production"
+    "types-publisher": "Microsoft/types-publisher#production",
+    "typescript": "^2.6.1"
+  },
+  "dependencies": {
+    "@types/jquery": "^3.2.16",
+    "jquery": "^3.1.1",
+    "typescript": "^2.6.1"
```

Comment:
This file shouldn't be changed.

Reply:
it is done

Reply:
It's still changed.

Reply:
I  just put the content like so

{
  "private": true,
  "name": "definitely-typed",
  "version": "0.0.1",
  "homepage": "https://github.com/DefinitelyTyped/DefinitelyTyped",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/DefinitelyTyped/DefinitelyTyped.git"
  },
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/DefinitelyTyped/DefinitelyTyped/issues"
  },
  "engines": {
    "node": ">=7.8.0"
  },
  "scripts": {
    "compile-scripts": "tsc -p scripts",
    "not-needed": "node scripts/not-needed.js",
    "test": "node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed",
    "lint": "dtslint types"
  },
  "devDependencies": {
    "@types/node": "^8.0.53",
    "dtslint": "github:Microsoft/dtslint#production",
    "types-publisher": "Microsoft/types-publisher#production",
    "typescript": "^2.6.1"
  },
  "dependencies": {
    "@types/jquery": "^3.2.16",
    "jquery": "^3.1.1",
    "typescript": "^2.6.1"
  }
}


It was so before

Reply:
Or what is missing? Can you help?

Reply:
I'm afraid it is not (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/package.json), you can just git reset this file